### PR TITLE
Localization: update base_map in local_pyramid_map module

### PR DESCRIPTION
**Pull request by tthhee**

### CODE REVIEW

File: modules/localization/msf/local_pyramid_map/base_map/BUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "localization_msf_base_map",
+    srcs = glob(["*.cc"]),
```
Comment (jinghaomiao) [inline modules/localization/msf/local_pyramid_map/base_map/BUILD]: One build target for each .cc file

File: modules/localization/msf/local_pyramid_map/base_map/BUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    hdrs = glob(["*.h"]),
+    linkopts = [
+        "-lz",
+        "-lopencv_core",
+        "-lopencv_highgui",
+        "-lopencv_imgproc",
+    ],
+    deps = [
+        "//modules/common/util",
```
Comment (michael4338) [inline modules/localization/msf/local_pyramid_map/base_map/BUILD]: sort the order of deps

File: modules/localization/msf/local_pyramid_map/base_map/base_map_node_config.h [hunk 3]
```diff
@@ -15,12 +15,14 @@
  *****************************************************************************/
 #pragma once
 
-#include "modules/localization/msf/local_map/base_map/base_map_config.h"
-#include "modules/localization/msf/local_map/base_map/base_map_node_index.h"
+#include <memory>
```
Comment (michael4338) [inline modules/localization/msf/local_pyramid_map/base_map/base_map_node_config.h]: separate the standard and project includes

File: modules/localization/msf/local_pyramid_map/base_map/BUILD (hunk 1 above)
Comment (tthhee) [reply modules/localization/msf/local_pyramid_map/base_map/BUILD]: fixed

File: modules/localization/msf/local_pyramid_map/base_map/BUILD (hunk 2 above)
Comment (tthhee) [reply modules/localization/msf/local_pyramid_map/base_map/BUILD]: fixed

File: modules/localization/msf/local_pyramid_map/base_map/base_map_node_config.h (hunk 3 above)
Comment (tthhee) [reply modules/localization/msf/local_pyramid_map/base_map/base_map_node_config.h]: fixed

**Pull request merged**